apps/backoffice-v2/cspell.config.cjs

```diff
@@ -0,0 +1,2 @@
+/** @type { import("@cspell/cspell-types").CSpellUserSettings } */
+module.exports = require('../../packages/config/cspell.base.cjs');
```

Comment:
CSpell will find a common root config file if it exists.

It will find:
- `.config/cspell.config.cjs`
- `cspell.config.cjs`

It is also possible to tell cspell to import the config from another location by adding the following to the `package.json` file:

`package.json`
```jsonc
{
   // ...
   "cspell": { "import": ["../../packages/config/cspell.base.cjs"] }
}
```